### PR TITLE
functions_framework_builder v0.3.1-dev

### PR DESCRIPTION
**Pull request by subfuzion**
Update `0.3.1-dev` functions_framework_builder changelog (pubspec.yaml has already been updated).

cc/@kevmoo I assume this is what you meant with [this comment](https://github.com/GoogleCloudPlatform/functions-framework-dart/pull/157#discussion_r549542586).

### CODE REVIEW

**Comment by kevmoo** [comment]
> Update `0.3.1-dev` functions_framework_builder changelog (pubspec.yaml has already been updated).
> 
> cc/@kevmoo I assume this is what you meant with [this comment](https://github.com/GoogleCloudPlatform/functions-framework-dart/pull/157#discussion_r549542586).

Yeah...although now that ff_0.3.1 is out, you should just remove the `-dev`.

File: functions_framework_builder/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## 0.3.1-dev
+
```
Comment (kevmoo) [inline functions_framework_builder/CHANGELOG.md:4]: Document the change: allow `functions_framework` versions xyz...

**Review by kevmoo — CHANGES_REQUESTED** [review]

File: functions_framework_builder/CHANGELOG.md (hunk 1 above)
Comment (subfuzion) [reply functions_framework_builder/CHANGELOG.md:4]: Should it or should it not match the [pubspec](https://github.com/GoogleCloudPlatform/functions-framework-dart/blob/main/functions_framework_builder/pubspec.yaml#L2). You were using `-dev` before. Or are you saying we need to bump *and* publish the builder to stay in sync with the ff version?

**Comment by subfuzion** [comment]
> > Update `0.3.1-dev` functions_framework_builder changelog (pubspec.yaml has already been updated).
> > cc/@kevmoo I assume this is what you meant with [this comment](https://github.com/GoogleCloudPlatform/functions-framework-dart/pull/157#discussion_r549542586).
> 
> Yeah...although now that ff_0.3.1 is out, you should just remove the `-dev`.

Should it match the current ffb pubspec: 0.3.1-dev, or no? (Or should that also be `0.3.1`?) And do we want to keep ff and ffb versions aligned?

**Comment by kevmoo** [comment]
@subfuzion – see https://github.com/GoogleCloudPlatform/functions-framework-dart/pull/160

**Comment by kevmoo** [comment]
Handled in https://github.com/GoogleCloudPlatform/functions-framework-dart/pull/160

**Pull request closed without merging**